docker-compose.yml

```diff
@@ -31,7 +29,8 @@ services:
         ports:
             - "3308:3306"
         volumes:
-            - /your/linguacafe/database/folder:/var/lib/mysql
+            - ./database:/var/lib/mysql
+        user: 1000:1000
```

Comment:
Will this `user: 1000:1000` affect already existing database folders?

Reply:
It was harmless when tested, but it is still an oversight that will be fixed.